docs/ref/models/fields.txt

```diff
@@ -1361,6 +1361,14 @@ can change the maximum length using the :attr:`~CharField.max_length` argument.
 The default form widget for this field is a
 :class:`~django.forms.ClearableFileInput`.
 
```

Comment:
We also need a release note in 5.1, something like:
```
* :class:`~django.db.models.ImageField` no longer re-reads the file from the
  storage backend after ``width_field`` and ``height_field`` are set. If
  your storage backend resizes images, the actual image's width and height will
  not match the auto-populated fields.
```
I think within Backwards incompatible changes in 5.1 and in Miscellaneous maybe

Reply:
I believe I have resolved this.